apps/cms.c

```diff
@@ -1458,10 +1459,14 @@ static CMS_ReceiptRequest
     } else {
         rct_from = NULL;
     }
```

Comment:
This seems completely superfluous. Could you remove it when you're touching this code?

Reply:
even more ugly is the function header:
```C
static CMS_ReceiptRequest
*make_receipt_request(STACK_OF(OPENSSL_STRING) *rr_to, int rr_allorfirst,
                      STACK_OF(OPENSSL_STRING) *rr_from)
```

Reply:
Agree it's superfluous.